### ISSUE TITLE
Celcius?

### ISSUE THREAD

**Issue by vBrolin** [jrowny/ESPHomeSpa#2]
Hi,
Trying to get temperature in C to work, getting F with just double the C value right now.
You said in the readme that it was hard coded, but I seem to struggle to find where, could you point it out and I could perhaps just hardcode my config for C instead?

**Comment by jrowny** [comment]
yes, I commented it out here: https://github.com/jrowny/ESPHomeSpa/blob/main/spa_reader.h#L210

I think the problem was that I was reading the state before I knew the spa state, but it seemed to not be correct when I was testing.

**Comment by martijnbuts** [comment]
> Hi, Trying to get temperature in C to work, getting F with just double the C value right now. You said in the readme that it was hard coded, but I seem to struggle to find where, could you point it out and I could perhaps just hardcode my config for C instead?


Hi, i have the same problem with the double C temperature. 
Did you solve the problem and if so can you share your config?

**Comment by vBrolin** [comment]
> > Hi, Trying to get temperature in C to work, getting F with just double the C value right now. You said in the readme that it was hard coded, but I seem to struggle to find where, could you point it out and I could perhaps just hardcode my config for C instead?
> 
> Hi, i have the same problem with the double C temperature. Did you solve the problem and if so can you share your config?

Created a fork of my bodge job of C-converting this, here you go:
https://github.com/vBrolin/ESPHomeSpa
